### ISSUE TITLE
Duplicate columns created in the configuration.stc after editing a column tip and description

### ISSUE THREAD

**Issue by wondie** [gltn/stdm#249]
Duplicate columns created in the configuration.stc after editing a column tip and description. Please see the stc created using the wizard. 

```
  <Entity editable="True" global="False" associative="False" description="Enumeration information" proxy="False" createId="True" supportsDocuments="False" shortName="Enumeration" name="do_enumeration">
   <Columns>
    <Column unique="False" tip="" minimum="-2147483648" maximum="2147483647" description="" index="False" name="id" searchable="True" TYPE_INFO="SERIAL" mandatory="False"/>
    <Column unique="False" tip="" minimum="2017-07-10" maximum="2050-12-13" description="" index="False" name="english_date_of_enumeration" searchable="False" TYPE_INFO="DATE" mandatory="False">
     <currentDate minimum="False" maximum="True"/>
    </Column>
    <Column unique="False" tip="" minimum="0" maximum="2147483647" description="" index="False" name="respondent" searchable="False" TYPE_INFO="FOREIGN_KEY" mandatory="False">
     <Relation name="fk_do_household_members_id_do_enumeration_respondent"/>
    </Column>
    <Column unique="False" tip="" minimum="0" maximum="2147483647" description="" index="False" name="gps_id_number" searchable="True" TYPE_INFO="LOOKUP" mandatory="False">
     <Relation name="fk_do_check_device_id_id_do_enumeration_gps_id_number"/>
    </Column>
    <Column unique="False" tip="" minimum="0" maximum="2147483647" description="" index="False" name="camera_id_number" searchable="True" TYPE_INFO="LOOKUP" mandatory="False">
     <Relation name="fk_do_check_device_id_id_do_enumeration_camera_id_number"/>
    </Column>
    <Column unique="True" tip="तथ्यांक संकलक को (थर, नाम) " minimum="0" maximum="50" description="तथ्यांक संकलक को (थर, नाम) " index="False" name="questionnaire_number" searchable="True" TYPE_INFO="VARCHAR" mandatory="True"/>
    <Column unique="True" tip="तथ्यांक संकलक को (थर, नाम) " minimum="0" maximum="50" description="तथ्यांक संकलक को (थर, नाम) " index="False" name="questionnaire_number" searchable="True" TYPE_INFO="VARCHAR" mandatory="True"/>

   </Columns>
  </Entity>
```
@gkahiu this is happening in master.

**Comment by gkahiu** [comment]
Issue fixed in this commit [2569f72](https://github.com/gltn/stdm/commit/2569f721179512750f73f1a3b42c36b2d8104981)

It was caused by a custom dictionary not re-indexing the position of items when an item is removed.

Please test and revert.

**Comment by wondie** [comment]
This issue is fixed.
